### ISSUE TITLE
how to run the code?

### ISSUE THREAD

**Issue by qzhai** [Sleepwalking/pytorch-softdtw#1]

**Comment by Sleepwalking** [comment]
Hello,

Say that you are training a sequence prediction model using the standard `nn.MSELoss` and there's a line like this in the training script,

```python
L = loss(output, target)
L.backward()
```

where `output` and `target` are matrices of shape `length x dimension`.

To use SoftDTW, just make it like this,
```python
softdtw = SoftDTW.apply
...
D = ((output.unsqueeze(0) - target.unsqueeze(1)) ** 2).mean(dim = 2)
L = softdtw(D, 0.1)
L.backward()
```
Note that depending on application you may want to normalize the DTW distance by sequence length.
